### ISSUE TITLE
InnerSource @ Zalando | innersource-howto

### ISSUE THREAD

**Issue by spier** [InnerSourceCommons/InnerSourcePatterns#394]
Zalando has published some tips about InnerSource on this public page:
https://opensource.zalando.com/docs/resources/innersource-howto/ (see [source](https://github.com/zalando/zalando.github.io))

There are further mentions of InnerSource at Zalando from as early as 2016 here:
* https://engineering.zalando.com/tags/innersource.html
* https://opensource.zalando.com/blog/2018/09/strengthens-innersource-strategy/

**Comment by lenucksi** [comment]
@hpdang knows more here.

**Comment by hpdang** [comment]
@lenucksi yes I can help but I'm not sure what is required in this ticket.

**Comment by spier** [comment]
@hpdang thanks for the quick reply, and nice to meet you.

This ticket is about figuring out what the latest status of InnerSource practices at Zalando is, given that the resources linked to above are pretty old.

From there we could see if extracting some InnerSource patterns out of Zalando's experience would be possible. We would then share those patterns in this repo and also at https://patterns.innersourcecommons.org once they are mature enough.

If you would like to chat more about this, I am happy to (I see you are also in the ISC Slack?).
